Dockerfile

```diff
@@ -25,6 +27,13 @@ FROM checkmarx/kics:alpine as kics
 # Get base image #
 ##################
 FROM python:3.9.7-alpine3.13
+FROM mikefarah/yq:latest as yq
+
+ARG YQ_VERSION='v4.14.2'
+ARG YQ_BINARY='yq_linux_amd64'
+
+# yq installation
+RUN wget -nv https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/${YQ_BINARY} -O /usr/bin/yq && chmod +x /usr/bin/yq
```

Comment:
I know you mentioned this was one of the reasons this is a draft PR. Do you already have another plan in mind for the next iteration? I am not sure what the best way to handle this is off the top of my head, but I can do some digging if nobody knows.

Reply:
If we replace `yq` by `PyYAML` there is no good reason to keep it here !